<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>腾飞 田径运动检测与分析系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>new_logo.png</normaloff>new_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F7F7F7</string>
  </property>
  <widget class="QWidget" name="topwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-140</x>
     <y>-30</y>
     <width>1600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>-150</x>
     <y>-80</y>
     <width>1621</width>
     <height>891</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-image: url(D:/Program Files (x86)/Study/NEW_JumpTest_1.0/public/resources/result.png);
    background-repeat: no-repeat;
    background-position: center;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAbdominalContractionScore">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; color: white; font-size: 24px;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHipExtensionScore">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; color: white; font-size: 24px;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="StartBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>480</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton  {
background: transparent; border: none
 
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Selet_hipBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton  {
background: transparent; border: none
 
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_img2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_img1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_speedPlot">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>130</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_video">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>521</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTakeOffScore">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; color: white; font-size: 24px;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelMeanVector">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; color: white; font-size: 24px;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_img3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Selet_takeoffBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton  {
background: transparent; border: none
 
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Selet_abdBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton  {
background: transparent; border: none
 
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SwitchBasePathBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>480</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton  {
background: transparent; border: none
 
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelAllScore">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>480</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; color: white; </string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Ca_img_but1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Ca_img_but2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Ca_img_but3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>topwidget</zorder>
  <zorder>tableWidget</zorder>
  <zorder>labelAbdominalContractionScore</zorder>
  <zorder>labelHipExtensionScore</zorder>
  <zorder>StartBtn</zorder>
  <zorder>Selet_hipBtn</zorder>
  <zorder>label_img2</zorder>
  <zorder>label_img1</zorder>
  <zorder>label_speedPlot</zorder>
  <zorder>label_video</zorder>
  <zorder>labelTakeOffScore</zorder>
  <zorder>labelMeanVector</zorder>
  <zorder>label_img3</zorder>
  <zorder>Selet_takeoffBtn</zorder>
  <zorder>Selet_abdBtn</zorder>
  <zorder>SwitchBasePathBtn</zorder>
  <zorder>labelAllScore</zorder>
  <zorder>Ca_img_but1</zorder>
  <zorder>Ca_img_but3</zorder>
  <zorder>Ca_img_but2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
